dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+# Created by Barbara Ellynes Zucchi Nobre Silva for the newsmap package: https://github.com/koheiw/newsmap
+
+AFRICA:
+  EAST:
+    'BI': [Burundi, Burúndi, República do Burúndi, Burundian*, 	Burundiense*, Burundinês*, Bujumbura]
```

Comment:
"República do Burúndi" is redundant because "Burúndi" matches the same string. Applies to almost all the "República do xxxx".

Reply:
I will update the file, taking out unnecessary repetitions.